photo/views.py

```diff
@@ -1,7 +1,47 @@
 from django.http import HttpResponse
-from django.shortcuts import render
+from rest_framework import generics
+from photo.models import Submission, User, Contest
+from photo.serializers import SubmissionSerializer, UserSerializer, ContestSerializer
+
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
 
 # Create your views here.
 
+
 def hello(request):
-  return HttpResponse("Hello Runtime")
+    return HttpResponse("Hello Runtime")
+
+
+class UserList(generics.ListCreateAPIView):
```

Comment:
Use ModelViewSet and replace all these methods

Reply:
done